espanol/jordiHurtado/inmortal/inmortal.md

```diff
@@ -1 +1,5 @@
-Y Jordi Hurtado creò una pociòn de inmmortalidad que terminò bebièndose èl.
\ No newline at end of file
+Y Jordi Hurtado creò una pociòn de inmmortalidad que terminò bebièndose èl.
+
+Tras dar buena cuenta de todo aquel ejército primero gracias a su inmortalidad, y tras otras múltples gestas y conquistas, Jordi estaba terriblemente aburrido.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.